src/saving/SaveHelper.cs

```diff
@@ -116,18 +116,36 @@ public static void LoadSave(string name)
     /// <summary>
     ///   Loads the save file with the latest write time
     /// </summary>
-    public static void QuickLoad()
+    /// <returns>False if the versions do not match</returns>
+    public static bool QuickLoad()
     {
         // TODO: is there a way to to find the latest modified file without checking them all?
         var save = CreateListOfSaves(SaveOrder.LastModifiedFirst).FirstOrDefault();
-
         if (save == null)
         {
             GD.Print("No saves exist, can't quick load");
-            return;
+            return true;
+        }
+
+        SaveInformation info;
+        try
+        {
+            info = global::Save.LoadJustInfoFromSave(save);
+        }
+        catch (Exception ex)
+        {
+            GD.PrintErr(ex.ToString());
```

Comment:
Add a short message before the exception message, like "Can't get save info for quick load: "